using-the-website/bond_frax.md

```diff
@@ -0,0 +1,7 @@
+# FRAX Bond
+
+FRAX bond was introduced on 16 June 2021. It allows you to trade [FRAX](https://www.coingecko.com/en/coins/frax) in exchange for discounted OHM on Olympus.
+
+## How to Bond
+
+Check out our [FRAX bond purchase guide](https://docs.olympusdao.finance/partnership/frax-partnership/bonds#frax-bond) for buying FRAX bond on Olympus.
```

Comment:
I find it kind of weird that we link to yet another page here. The DAI bond has all instructions in this page and doesn't redirect somewhere else. Do you think it makes sense to just move the instructions here? Or what were your thoughts on this?

Reply:
The initial thought was that users may go to either the Frax Partnership section or Using the Website section to find the bond guide. So being redundant - keeping both pages - seemed like a good idea.

Reply:
But since the [staking of OHM-FRAX LP token](https://docs.olympusdao.finance/partnership/frax-partnership/staking_lp) has been deprecated, this page should be removed soon. So I think it makes sense to move the bond instructions under Frax Partnership to Using the Website section.

Reply:
I will close this PR first while working on the pages.